PlutoHelperAgent/PreferencesWindowController.m

```diff
@@ -207,4 +135,13 @@ - (IBAction)saveClicked:(id)sender {
     [super close];
     
 }
+
+- (IBAction)testClicked:(id)sender {
+    [SharedFunctions logout_of_project_server];
```

Comment:
how will this behave if you're not logged into the server when it's run?

Reply:
If fails to log out, obviously, but the programme still works, I have tested it. It just throws errors that do not break anything.